qemu/tests/cfg/readonly_floppy.cfg

```diff
@@ -1,12 +1,13 @@
 - readonly_floppy:
     virt_test_type = qemu
     type = readonly_floppy
+    start_vm = no
```

Comment:
if you want to dynamically change the params after test starts and then start the vm, it may better to use `not_preprocess = yes`

Reply:
If use \"not_preprocess = yes\",  the changed pre_command not run in \"env_process.preprocess\". And need to run \"env_process.process_command\" before \"env_process.preprocess\", I think it is duplicate. So I think \"start_vm = no\" is ok. Thank you.